src/meltano/core/select_service.py

```diff
@@ -39,19 +39,22 @@ def current_select(self):
         plugin_settings_service = PluginSettingsService(self.project, self.extractor)
         return plugin_settings_service.get("_select")
 
-    async def load_catalog(self, session):
+    async def load_catalog(self, session, refresh=False):
```

Comment:
What do you think about adding tests for this in `tests/meltano/cli/test_select.py`?

Reply:
I think I'm a little out of my depth on this. In order to test, I think I'd need a basic tap that would dynamically build a catalog from a data source, right? Like a tap-json where I modify the json in between runs? Just using tap-mock (which I can't find the code for anyway) wouldn't test this.

Am I overcomplicating this?

Reply:
See https://github.com/andyoneal/meltano/pull/1

Reply:
thanks. merged.